Add blogpost for podcast episode 4 (#160)
publiccodenet/blog: _posts/2021-03-04-episode-four-of-lets-talk-about-public-code.md

---
title: "Episode four of Let's talk about public code! - Lea Hemetsberger from Open & Agile Smart Cities"
date: 2021-03-04
author: Jan Ainali
type: blogpost
excerpt: The fourth episode of our podcast is available
categories:
  - news
---

# Episode four of Let's talk about public code!

> In the fourth episode we talked to Lea Hemetsberger from [Open & Agile Smart Cities](https://oascities.org/).

Reviewer: ```suggestion
In the fourth episode we talked to Lea Hemetsberger from [Open & Agile Smart Cities](https://oascities.org/) (OASC).
```

> We discussed how they are collaborating with cities worldwide to support their digital transformation journey.

Reviewer: ```suggestion
We discussed how OASC collaborates with cities worldwide to support their digital transformation journey.
```

> We also talked about one of their latest resources, the [CITYxCITY Academy](https://citibycity.academy).

Reviewer: ```suggestion
We also talked about one of their latest resources, the [CITYxCITY Academy](https://citybycity.academy).
```

You can find the podcast at [podcast.publiccode.net](https://podcast.publiccode.net/e/4-lea-hemetsberger-open-agile-smart-cities/) or by searching in your favorite podcast app.

[![Lea, Jan and Alba chatting in a video chat grid]({{site.url}}/assets/screenshot-episode-4.jpg)](https://www.youtube.com/watch?v=5iq1Mqwah7g)

## Where can I find the video?

The video is available in a number of places:

- [Youtube](https://www.youtube.com/watch?v=5iq1Mqwah7g)
- [Periscope](https://www.pscp.tv/w/1DXxyRyyWnZKM)
- [Facebook](https://www.facebook.com/publiccodenet/videos/430134378050506/)

## What's next?

> The next episode will be in April, but date and guest is not set yet. If you have any suggestions for public coders we should talk to, reach out to us on [Twitter](https://twitter.com/publiccodenet) or add a tip in [an issue in our Projects repository](https://github.com/publiccodenet/projects/issues/new).

Reviewer: ```suggestion
The next episode will be in April, but date and guest are not set yet. If you have any suggestions for public coders we should talk to, reach out to us on [Twitter](https://twitter.com/publiccodenet) or comment in [an issue in our Projects repository](https://github.com/publiccodenet/projects/issues/new).
```
